sources/LibertinusSerif-Italic.sfd

```diff
@@ -20,6 +20,7 @@ OS2Version: 0
 OS2_WeightWidthSlopeOnly: 0
 OS2_UseTypoMetrics: 1
 CreationTime: 1155510122
+ModificationTime: 1727117497
```

Comment:
This needs to be normalized again to get rid of this and a few other artifacts, but I can take care of that if you don't get too it before I merge this (which should be soon, and I'll probably fast-track a patch release so folks playing around with the new Hebrew stuff have it to work on).